### PR TITLE
[#3740] Update dotnet samples to target Net 6

### PR DESCRIPTION
**Pull request by erquirogasw**
Addresses # 3740
#minor

## Description
This PR updates the BotBuilder samples to target .NET6

## Specific Changes
- Updated the following samples to target .NET6
  - Console-echo
  - Echo-bot
  - Welcome-user
  - Multi-turn-prompt
  - Using-cards
  - Using-adaptive-cards
  - Suggested-actions
  - QnAmaker
  - CustomQABot
  - Core-bot
  - Npl-with-orchestrator
  - Handling-attachments
  - Proactive-messages
  - Multilingual-bot
  - Custom-dialogs
  - Corebot-app-insights
  - Facebook-events
  - Bot-authentication-msgraph
  - Message-reaction
  - Timex-resolution
  - Scaleout
  - Completx-dialog
  - Prompt-users-for-input
  - State-management
  - Teams-auth
  - Inspection
  - CustomQABot-all-features
  - QnAmaker-all-features
  - Teams-messaging-extensions-search
  - Teams-messaging-extensions-action
  - Teams-messaging-extensions-search-auth-config
  - Teams-messaging-extensions-action-preview
  - Teams-task-module
  - Teams-link-unfurling
  - Teams-file-upload
  - Teams-conversation-bot
  - Teams-start-new-thread-in-channel
  - Skills-simple-bot-to-bot
  - Skills-skilldialog
  - Skills-sso-cloudadapter
- README files updated to target .NET6

## Testing

_Successful build_
image (74)

### CODE REVIEW

File: samples/csharp_dotnetcore/01.console-echo/README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ This sample shows a simple echo bot and demonstrates the bot working as a consol
 
 ## Prerequisites
 
-- [.NET Core SDK](https://dotnet.microsoft.com/download) version 3.1
+- [.NET Core SDK](https://dotnet.microsoft.com/download) version 6.0
```
Comment (ceciliaavila) [inline samples/csharp_dotnetcore/01.console-echo/README.md]: Net 6 is not Core or Framework, just Net 6. If you see the changes I did for the generators, I left this as:
```suggestion
- [.NET SDK](https://dotnet.microsoft.com/download) version 6.0

```

**Review by ceciliaavila — CHANGES_REQUESTED** [review]
These are too many files for one PR. After we finish testing the samples, I think we should split the samples into two or three PRs.

**Comment by erquirogasw** [comment]
> These are too many files for one PR. After we finish testing the samples, I think we should split the samples into two or three PRs.

Is it ok if I create three PRs part 1, part 2 and part 3?

**Comment by ceciliaavila** [comment]
> > These are too many files for one PR. After we finish testing the samples, I think we should split the samples into two or three PRs.
> 
> Is it ok if I create three PRs part 1, part 2 and part 3?

We could split the samples following the categories we used for testing. You can have more than one category per PR, depending on the number of files.

**Pull request closed without merging**